app/controllers/items/index.js

```diff
@@ -74,6 +83,10 @@ export default Ember.Controller.extend(SearchMixin, {
         .then(results => {
           return results;
         });
+    },
+
+    clearSearch() {
```

Comment:
@shreyas098 where are you using this action ?

Reply:
For the cross button which appears on the search tab